### PR TITLE
Update orchestrator to 3.24

### PR DESCRIPTION
**Pull request by saberduck**

### CODE REVIEW

File: its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java [hunk 1]
```diff
@@ -66,39 +65,34 @@ public static SonarScanner createSonarScanner() {
...
+
   @CheckForNull
-  static Component searchComponent(Orchestrator orchestrator, String projectKey, String componentKey) {
-    List<Component> components = newWsClient(orchestrator).components().tree(
-      new TreeWsRequest()
-        .setBaseComponentKey(projectKey)
-        .setQuery(componentKey))
+  static Components.Component searchComponent(Orchestrator orchestrator, String componentKey) {
+    List<Components.Component> components = newWsClient(orchestrator).components()
+      .search(new SearchRequest().setQ(componentKey).setQualifiers(singletonList("FIL")))
```
Comment (andrei-epure-sonarsource) [inline its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java]: please add a comment for what `"FIL"` means

**Comment by saberduck** [comment]
merged in #110

File: its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java (hunk 1 above)
Comment (andrei-epure-sonarsource) [reply its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java]: not a big thing, but `"FIL"` is a little bit cryptic to me. Do we have documentation for it somewhere?

Comment (saberduck) [reply its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java]: I added the documentation, it was merged in another PR https://github.com/SonarSource/sonar-html/blob/master/its/plugin/src/test/java/com/sonar/it/web/HtmlTestSuite.java#L91 

this is document in the webservice documentation, but I agree it is bit cryptic

**Pull request closed without merging**